docs(ter): review luiza products

ldecarvalho-doc opened the pull request:
Made a couple of small changes to text and form for the docs of my products.

Codelax commented on docs/resources/rdb_read_replica.md:
```diff
@@ -85,48 +85,48 @@ The following arguments are supported:
-    - `private_network_id` - (Required) UUID of the private network to be connected to the read replica.
-    - `service_ip` - (Optional) The IP network address within the private subnet. This must be an IPv4 address with a CIDR notation. If not set, The IP network address within the private subnet is determined by the IP Address Management (IPAM) service.
-    - `enable_ipam` - (Optional) If true, the IP network address within the private subnet is determined by the IP Address Management (IPAM) service.
+- `private_network` - (Optional) Create an endpoint in a Private Netork.
+  - `private_network_id` - (Required) UUID of the Private Netork to be connected to the Read Replica.
```
```suggestion
- `private_network` - (Optional) Create an endpoint in a Private Network.
  - `private_network_id` - (Required) UUID of the Private Network to be connected to the Read Replica.
```

Codelax commented on docs/resources/rdb_read_replica.md:
```diff
@@ -85,48 +85,48 @@ The following arguments are supported:
+  - `port` - TCP port of the endpoint.
+  - `name` - Name of the endpoint.
+  - `hostname` - Hostname of the endpoint. Only one of IP and hostname may be set.
+- `private_network` - List of Private Netorks endpoints of the Read Replica.
```
```suggestion
- `private_network` - List of Private Networks endpoints of the Read Replica.
```

Codelax commented on docs/resources/rdb_instance.md:
```diff
@@ -155,75 +154,75 @@ and if you are using `bssd` storage, you should increase the volume size before
-~> **Important:** Updates to `private_network` will recreate the Instance's endpoint
-  
-~> **NOTE:** Please calculate your host IP using [cidrhost](https://developer.hashicorp.com/terraform/language/functions/cidrhost). Otherwise, let IPAM service
+~> **Note** You can calculate your host IP using [cidrhost](https://developer.hashicorp.com/terraform/language/functions/cidrhost). Otherwise, let IPAM service
```
```suggestion
-> **Note** You can calculate your host IP using [cidrhost](https://developer.hashicorp.com/terraform/language/functions/cidrhost). Otherwise, let IPAM service
```
Let's use blue color as the other notes ([format](https://developer.hashicorp.com/terraform/registry/providers/docs#callouts))

Codelax commented on docs/resources/iam_policy.md:
```diff
@@ -5,9 +5,9 @@ page_title: "Scaleway: scaleway_iam_policy"
 # Resource: scaleway_iam_policy
 
-Creates and manages Scaleway IAM Policies. For more information, see [the documentation](https://www.scaleway.com/en/developers/api/iam/#policies-54b8a7).
+Creates and manages Scaleway IAM Policies. For more information refer to the [IAM API documentation](https://www.scaleway.com/en/developers/api/IAM/#policies-54b8a7).
```
```suggestion
Creates and manages Scaleway IAM Policies. For more information refer to the [IAM API documentation](https://www.scaleway.com/en/developers/api/iam/#policies-54b8a7).
```

Codelax commented on docs/resources/iam_policy.md:
```diff
@@ -5,9 +5,9 @@ page_title: "Scaleway: scaleway_iam_policy"
+Creates and manages Scaleway IAM Policies. For more information refer to the [IAM API documentation](https://www.scaleway.com/en/developers/api/IAM/#policies-54b8a7).
 
-~> You can find a detailed list of all permission sets available at Scaleway in the permission sets [reference page](https://www.scaleway.com/en/docs/identity-and-access-management/iam/reference-content/permission-sets/).
+-> You can find a detailed list of all permission sets available at Scaleway in the permission sets [reference page](https://www.scaleway.com/en/docs/identity-and-access-management/IAM/reference-content/permission-sets/).
```
```suggestion
-> You can find a detailed list of all permission sets available at Scaleway in the permission sets [reference page](https://www.scaleway.com/en/docs/identity-and-access-management/iam/reference-content/permission-sets/).
```

Codelax commented:
There are multiple lint errors. All are due to the unexpected indentation for list elements.
```
docs/data-sources/billing_invoices.md:39:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:40:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:41:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:42:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:43:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:44:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:45:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:46:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:47:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:48:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:49:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:50:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:51:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:52:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:53:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/data-sources/billing_invoices.md:54:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/iam_policy.md:110:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/iam_policy.md:111:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/iam_policy.md:115:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:179:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:180:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:181:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:205:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:206:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:207:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:209:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:210:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:211:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:212:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:213:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:215:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:216:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:217:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:218:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_instance.md:219:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:93:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:94:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:95:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:114:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:115:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:116:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:117:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:118:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:120:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:121:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:122:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:123:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:124:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/rdb_read_replica.md:125:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/redis_cluster.md:183:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/redis_cluster.md:184:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/redis_cluster.md:185:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/redis_cluster.md:189:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/redis_cluster.md:190:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/tem_domain.md:149:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/tem_domain.md:150:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/tem_domain.md:151:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/tem_domain.md:152:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
docs/resources/tem_domain.md:153:1 MD007/ul-indent Unordered list indentation [Expected: 4; Actual: 2]
```

remyleone commented:
I can indeed reproduce the issue.
@ldecarvalho-doc could you install `brew install markdownlint-cli2` and do a `markdownlint-cli2 docs`?
It will help you reproduce locally the issue that is happening in the CI

ldecarvalho-doc commented:
> I can indeed reproduce the issue. @ldecarvalho-doc could you install `brew install markdownlint-cli2` and do a `markdownlint-cli2 docs`? It will help you reproduce locally the issue that is happening in the CI

Hey, I finally found what was causing the issues and fixed them.
Thanks!
Seems like all checks passed, but it's telling me the branch is out-of-date with master, even though I rebased like 20min ago... should I do it again?
